### DISCUSSION TITLE
Dexcom One+ with Build your own Dexcom app / DexcomAppxDrip

### DISCUSSION THREAD

**Discussion by random-guy-12345** [NightscoutFoundation/xDrip#3391] [Q&A]
Any ideas if one of the versions existing in the form will work with Dexcom One+?
If not, are there plans to support Dexcom One+ with "Build your own Dexcomm app" in the future?
I have an incompatible phone and I wanted to compare Libre2 with Dexcom One+ (wearing both at the same time), but since I've been using Libre2 so far quite succesfully with XDrip, I wanted to keep it connected to XDrip and have the Dexcom on the official app for now.

**Comment by Navid200** [comment]
You are in the xDrip discussions.
xDrip supports Dexcom One+.

Please let me know if you have any questions about xDrip.

Or,  you can use a different forum for questions about other apps.

**Reply by random-guy-12345** [reply]
Is this a separate project https://navid200.github.io/xDrip/docs/DexcomAppxDrip.html?
I thought it's a part of XDrip, as well as using dexcom follow to get readings to XDrip (through the modified app) without internet is a part of XDrip.

In case it's a separate project, can you please share a link to discussions about it?

**Reply by michaelrennie17-droid** [reply]
How do you get xdrip+ to start a new sensor when there is no transmitter code?

**Reply by Navid200** [reply]
@michaelrennie17-droid Please open a new discussion and ask your question and I will help if I can.
